tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java

```diff
@@ -1,10 +1,10 @@
 package uk.gov.di.txma.eventprocessor.bdd;
 
+import io.cucumber.datatable.DataTable;
```

Comment:
remove this file as i think this is from denny's branch. not relevant to this pr